chapters/package-information.md

```diff
@@ -629,7 +629,7 @@ EXAMPLE 2 RDF: Property `spdx:filesAnalyzed` in class `spdx:Package`
 
 ### 7.9.1 Description
 
-This field provides an independently reproducible mechanism identifying specific contents of a package based on the actual files (except the SPDX document itself, if it is included in the package) that make up each package and that correlates to the data in this SPDX document. This identifier enables a recipient to determine if any file in the original package (that the analysis was done on) has been changed and permits inclusion of an SPDX document as part of a package. The metadata for the package verification code field is shown in Table 21.
+This field provides an independently reproducible mechanism that permits unique identification of a package and the files contained within it (except the SPDX document itself, if it is included in the package). The metadata for the package verification code field is shown in Table 21. For more specific use cases and examples of how to apply the package verification code field, see [Annex K.3 Verifying SPDX Packages](how-to-use.md#k3-verifying-spdx-packages).
```

Comment:
Suggest changing `This field provides an independently reproducible mechanism that permits unique identification of a package` to `This field provides an independently reproducible mechanism that permits unique identification of the contents a package`.  The reason for this change is the SPDX Package is defined not only by it's content but also the metadata associated with the content.  One example is that a supplier can choose to distribute the exact same content under different licenses to different downstream packages.  In SPDX these would be considered different packages even though the content is the same.  To uniquely identify the package, you would need to include the metadata as well as the content in the algorithm for the fingerprint.

Reply:
@goneall does this work? `This field provides an independently reproducible mechanism that permits unique identification of the files contained with a package (except the SPDX document file itself, if it is included in the package).` I would like to keep the clarification about the SPDX document itself not being included because that is a driver for having the two different PVC and PCS fields.

Reply:
@rnjudge - Your suggestion is fine - but if we want to be completely accurate, we could say `This field provides an independently reproducible mechanism that permits unique identification of the contents of the files contained with a package (except the SPDX document file itself, if it is included in the package).`